machine_learning_hep/utilities_validation.py

```diff
@@ -30,6 +30,44 @@ def fillvalidationvsmult(dfevt, dfevtevtsel, df_reco):
     Create a TH3F histogram and fill it with three variables from a dataframe.
     """
     _ = len(df_reco)
+    hlist = []
+    def makenfill(nx, xl, xh, ny, yl, yh, X, Y):
+      hn = "h_" + X + "_vs_" + Y
```

Comment:
Just a thing which might be helpful/fancy: Python has so called formatted strings and you can make them putting an f before the quotation marks, like
`hn = f"f_{X}_vs_{Y}"`

Reply:
Indeed, thanks, this will be useful